Makefile

```diff
@@ -24,11 +24,8 @@ CSRC=\
 	src/contrib/bcrypt/crypt_blowfish.c\
 	src/contrib/bcrypt/crypt_gensalt.c\
 	src/contrib/bcrypt/wrapper.c\
-	src/contrib/sqlite/sqlite3.c\
```

Comment:
This and the deletion of `src/contrib/sqlite/sqlite3*.h` cause problems for the Linux builds.

Reply:
fixed